appendix/migrate-to-postgresql.rst

```diff
@@ -57,42 +42,9 @@ Please also have a look at :doc:`/appendix/configure-database-server`.
 
       .. include:: /install/includes/postgres-dependencies.rst
 
-Database Credentials
---------------------
-
-Get the credentials Zammad is currently using to access your MySQL/MariaDB
-database from ``config/database.yml``. You will need them later.
-
-Now adjust the configuration file to fill in the credentials for your new
-PostgreSQL server. Use ``postgresql`` as ``adapter``.
 
-
-.. include:: /install/includes/postgres-permissions.rst
```

Comment:
This is actually a very important hint for the `rake db:create` part.
Removing this may leave people "not knowing what to do or why it fails" depending on the systems configuration.

As this part here expects the user to install postgresql and doesn't tell anything on "how to create the zammad db user" (which is fine because we expect the admin to know), the process will most likely fail. The default behavior of postgresql is to not allow database creation for normal users unless stated other wise.